src/app/core/components/header/header.component.ts

```diff
@@ -12,4 +13,8 @@ export class HeaderComponent {
 	logout() {
 		this.logOutEvent.emit();
 	}
+
+	getAppVersion() {
+		return packageInfo.version;
+	}
```

Comment:
I'd probably favour a readonly variable, meaning no need for a method and or test. e.g.

```
import {version} from '../../../../../package.json';

export class HeaderComponent {
 protected readonly version = version;
```